### PR TITLE
PXB-3098 : Remove --stats in 8.1

### PR DESCRIPTION
**Pull request by satya-bodapati**
The stats feature was introduced when server didn't expose the free
page information and the index stats information.

Since the same information is available from server, we will remove
the stats feature.

### CODE REVIEW

**Comment by satya-bodapati** [comment]
=============================================================================
Spent 21411 of 1397 seconds executing testcases

SUMMARY: 365 run, 296 successful, 67 skipped, 2 failed

Skipped tests:  binlog.bug1250375 pagetracking.encryption_after_checkpoint pagetracking.validate_tracking_start_lsn keyring.basic_encrytion keyring.innodb_encryption_mix_plugins keyring.innodb_keyring_kmip_component keyring.innodb_keyring_kms_component keyring.innodb_keyring_plugin_to_component keyring.innodb_keyring_vault_auto_v1 keyring.innodb_keyring_vault_v1 keyring.innodb_keyring_vault_v2 keyring.innodb_keyring_vault_auto_v2 keyring.keyring_encryption keyring.pxb-1793 keyring.pxb-2429 keyring.shared_tablespace_encryption xbcloud.backup_incomplete xbcloud.basic_operations xbcloud.ec2_instance_profile xbcloud.fifo xbcloud.redact_password xbcloud.storage_class rocksdb.compress_encrypt rocksdb.compress rocksdb.cross_engine_trx rocksdb.datadir rocksdb.ddl rocksdb.local_max_age rocksdb.local rocksdb.log_status rocksdb.stream_compress_encrypt rocksdb.stream_compress rocksdb.stream rocksdb.wal_dir_abs_archive rocksdb.wal_dir_abs rocksdb.wal_dir_rel main.backup_locks main.bug1192834-purge main.bug1192834-rollback main.bug1227240 main.bug1347698 main.bug1394493 main.bug1415191 main.bug1418584 main.bug1475487 main.bug1533722 main.bug382742 main.bug759225 main.bug976945 main.bug983720_galerainfo main.bug983720_lrudump main.distribution_message main.ib_doublewrite main.ib_lru_dump_basic main.ib_lru_dump_rsync main.ib_lru_dump_stream main.innodb_fast_checksum main.innodb_log_checksum_algorithm main.ib_part_tf_myisam main.instant_columns_8029 main.instant_columns main.pxb-1824 main.pxb-1914 main.pxb-2357 main.xb_apply_archived_logs main.xb_galera_info main.xb_galera_sst

Failed tests:  main.xb_stats_datadir main.xb_stats


--
I removed these tests but make doesn't delete the test files.

File: storage/innobase/xtrabackup/src/xtrabackup.cc [hunk 1]
```diff
@@ -783,10 +782,6 @@ struct my_option xb_client_options[] = {
     {"backup", OPT_XTRA_BACKUP, "take backup to target-dir",
      (G_PTR *)&xtrabackup_backup, (G_PTR *)&xtrabackup_backup, 0, GET_BOOL,
      NO_ARG, 0, 0, 0, 0, 0, 0},
-    {"stats", OPT_XTRA_STATS,
```
Comment (altmannmarcelo) [inline storage/innobase/xtrabackup/src/xtrabackup.cc:786]: OPT_XTRA_STATS - we can get rid of this enum entry.

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]
https://github.com/percona/percona-xtrabackup/commit/73977bce174afa7fb92098bb2065af933ac12c07 - Do we need to revert this?

Can you run this against the new 8.1 Jenkins?

**Comment by altmannmarcelo** [comment]
One further request, can you add a proper commit message :)

File: storage/innobase/xtrabackup/src/xtrabackup.cc (hunk 1 above)
Comment (satya-bodapati) [reply storage/innobase/xtrabackup/src/xtrabackup.cc:786]: good catch. removed

**Comment by satya-bodapati** [comment]
> [73977bc](https://github.com/percona/percona-xtrabackup/commit/73977bce174afa7fb92098bb2065af933ac12c07) - Do we need to revert this?
> 
> Can you run this against the new 8.1 Jenkins?

I think the doublewrite mode has some implications. Removing this change enables doublwrite buffer during prepare.
The change is at common location.

Please see my comments at https://jira.percona.com/browse/PXB-2879. lets enable it as part of PXB-2879.

**Comment by satya-bodapati** [comment]
https://pxb.cd.percona.com/view/PXB%208.1/job/percona-xtrabackup-8.1-multijob/10/

**Comment by satya-bodapati** [comment]
Finally a successful run: https://pxb.cd.percona.com/view/PXB%208.1/job/percona-xtrabackup-8.1-multijob/18/

**Comment by satya-bodapati** [comment]
@altmannmarcelo I see the usual failures only.

**Review by altmannmarcelo — APPROVED** [review]
LGTM

**Pull request merged**